watsapp/toPDF.py

```diff
@@ -0,0 +1,136 @@
+from fpdf import FPDF
```

Comment:
Consider the best structure for your project. Think about what are the libraries, and what the main application. In Recommenders, it made sense for us to have notebooks, maybe it doesn't make sense in your case. But whatever you decide, make sure it is simple and easy to understand.

FYI @KhaledAkel @adnanbhanji @smorenolasa1 @beaamoo